### DISCUSSION TITLE
DigiLocker integration with RC (for SSO)

### DISCUSSION THREAD

**Discussion by rhwarrier** [Sunbird-RC/community#399] [Ideas & Enhancements]
Initiating a thread for discussion around DigiLocker integration - this is towards enabling easy on boarding for users.

1. DigiLocker SSO onto an Issuer portal will allow the user to take advantage of seamless 1-click onboarding (onto the issuer portal)
2. Registering the issuer of DigiLocker will allow the end user to access their credentials via the DigiLocker
3. Presence of the credentials on DigiLocker also means that users can easily share credentials from multiplie issuers from one common point that provides a consolidated view to the user.

 Please add notes as necessary, as well work towards a design document. Thanks 

@parthlawate  @tejash-jl  @surendrasinghs

**Comment by Renuka-S** [comment]
@snehal0904 @coolbung Could you please share the design document for this & current state of implementation? cc: @parthlawate @surendrasinghs
